optaplanner-quickstarts/spring-boot-school-timetabling/src/main/java/com/example/schooltimetabling/domain/Lesson.java

```diff
@@ -61,6 +61,12 @@ public Lesson(String subject, String teacher, String studentGroup) {
         this.studentGroup = studentGroup.trim();
     }
 
+    public Lesson(String subject, String teacher, String studentGroup, Timeslot timeslot, Room room) {
```

Comment:
@ge0ffrey The NPE you've reported is caused right here. You're not setting `id`, and therefore `@PlanningId` is null, therefore NPE in CS-D.

Reply:
In my opinion, this is not a bug. Well, not in CS-D anyway.

Reply:
That should give a better error message.

Reply:
Fixed, but we should discuss proposal C) to reduce the boilerplate for testing.